src/main/webapp/js/map_loader.js

```diff
@@ -0,0 +1,24 @@
+let map;
+function createMap(){

```

Comment:
For this function I think your method comment should only need to mention that the map is placed in a map element, and that is has an event listener